assets/app/scripts/directives/resources.js

```diff
@@ -45,9 +53,27 @@ angular.module('openshiftConsole')
     return {
       restrict: 'E',
       scope: {
-        pods: '='
+        pods: '=',
+        projectName: '@?' //TODO optional for now
       },
-      templateUrl: 'views/_pods.html'
+      templateUrl: 'views/_pods.html',
+      controller: function($scope) {
+        $scope.phases = [
+          "Failed",
+          "Pending",
+          "Running",
+          "Succeeded"
```

Comment:
Updated to warning style.  I'm going to see if I can do a bootstrap tooltip with the reason for the warning.